cmd/keys/export/export.go

```diff
@@ -0,0 +1,57 @@
+package export
+
+import (
+	"fmt"
+	"github.com/dymensionxyz/roller/cmd/consts"
+	"github.com/dymensionxyz/roller/cmd/utils"
+	"github.com/spf13/cobra"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export <key-id>",
+		Short: "Exports the private key of the given key id",
```

Comment:
what the expected key_id here? should be printed

Reply:
fixed